<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QWidget" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>771</width>
           <height>560</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="mainMenuFrame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="1">
                <layout class="QGridLayout" name="gridLayout_5">
                 <property name="sizeConstraint">
                  <enum>QLayout::SizeConstraint::SetFixedSize</enum>
                 </property>
                 <item row="1" column="1">
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="2">
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="musicButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>30</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="checkable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>13</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="2" column="3">
                <widget class="QLabel" name="welcomeLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>36</pointsize>
                   <bold>true</bold>
                   <strikeout>false</strikeout>
                  </font>
                 </property>
                 <property name="text">
                  <string>CONNECT 4</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="3">
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="sizeConstraint">
                  <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
                 </property>
                 <item row="3" column="1">
                  <widget class="QPushButton" name="aboutButton">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="text">
                    <string>About</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="6" column="1">
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="2" column="1">
                  <spacer name="verticalSpacer_7">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>10</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="newGameButton">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="text">
                    <string>New Game</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <spacer name="verticalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="5" column="1">
                  <widget class="QPushButton" name="quitButton">
                   <property name="minimumSize">
                    <size>
                     <width>200</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>200</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="text">
                    <string>Quit</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="2">
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="4" column="1">
                  <spacer name="verticalSpacer_9">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Minimum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>10</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="3" column="4">
                <layout class="QGridLayout" name="gridLayout_4">
                 <property name="sizeConstraint">
                  <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
                 </property>
                 <item row="0" column="1">
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>90</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="rankingButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="toolTip">
                    <string/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>45</width>
                     <height>45</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="4" column="1">
                  <spacer name="verticalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Policy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="3" column="1">
                  <widget class="QPushButton" name="optionsButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="toolTip">
                    <string/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>45</width>
                     <height>45</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="historyButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="toolTip">
                    <string/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>45</width>
                     <height>45</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="4">
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="0" column="1">
                  <widget class="QPushButton" name="profileButton">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::FocusPolicy::TabFocus</enum>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>../media/images/icons/dark-theme/profile.png</normaloff>../media/images/icons/dark-theme/profile.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>45</width>
                     <height>45</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Orientation::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="1" column="1">
                  <spacer name="verticalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Orientation::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="0" column="3">
                <spacer name="verticalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Policy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>13</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="2" column="5">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Policy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>10</width>
                   <height>34</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
